terraform/production/main.tf

```diff
@@ -33,3 +33,14 @@ module "networking" {
   vpc_cidr_block                          = "10.0.0.0/16"
   vpc_flow_cloudwatch_log_expiration_days = 90
 }
+
+module "rds" {
+  source = "../modules/rds"
+
+  prefix               = "core-stag"
+  allocated_storage    = 100
+  db_subnet_group_name = module.networking.private_subnet_group_name
+  instance_class       = "db.t3.small"

```

Comment:
PaaS uses `small-ha-13` for prod and this looks like it is actually `db.t3.small` in AWS so I've used this here (we check this using `cf marketplace -e postgres`)